model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -706,6 +714,16 @@ message PubSubWritePayload {
   string id_attribute = 3;
 }
 
+// Payload for GroupIntoBatches composite transform.
+message GroupIntoBatchesPayload {
+
+  // (Required) Max size of a batch.
+  int64 batch_size = 1;
+
+  // (Optional) Max duration a batch is allowed to be cached in states.
+  int64 max_buffering_duration_millis = 2;
```

Comment:
It seems like `max_buffering_duration_millis is None` and `max_buffering_duration_millis  == 0` means the same. right? You may want to add a comment about  what will happen if `max_buffering_duration_millis` is not specified or `max_buffering_duration_millis` is set to 0. Someone might think `max_buffering_duration_millis = 0` or `max_buffering_duration_millis is None` will make the transform output elements as soon as possible.

Reply:
Were you suggesting adding a comment here in the runner proto or in GroupIntoBatches transform? I added some comments around here when changing the API: https://github.com/apache/beam/blob/c1541796435cbd2ee775b6c89ec628f8564a3bcd/sdks/python/apache_beam/transforms/util.py#L767